SokolTextGame/Weapons.cs

```diff
@@ -4,26 +4,31 @@ public interface IWeapon
     {
         string Name { get; set; }
         public string Description { get; }
+        public int WeaponDamage { get; set; }
     }
 
     public class BareFists : IWeapon
     {
         public string Description { get; } = "Bare fists. What else could it be?\n";
         public string Name { get; set; } = "bare fists";
+        public int WeaponDamage { get; set; } = 10;
     }
     public class Axe : IWeapon
     {
         public string Description { get; } = "There is always something to do with an axe.\nYou can split firewood, or you can split a skull.\n";
         public string Name { get; set; } = "axe";
+        public int WeaponDamage { get; set; } = 25;
```

Comment:
Make at least one of your weapons deal more damage so that the player's damage is not always exactly equal to the monster's.